src/Strategy/Ruby/GemfileLock.hs

```diff
@@ -89,9 +91,9 @@ toDependency pkg = foldr applyLabel start
         }
 
     applyLabel :: GemfileLabel -> Dependency -> Dependency
-    applyLabel (GemfileVersion ver) dep = dep{dependencyVersion = Just (CEq ver)}
+    applyLabel (GemfileVersion ver) dep = dep{dependencyVersion = (dependencyVersion dep) <|> Just (CEq ver)}
```

Comment:
[prefer]: 
```suggestion
    applyLabel (GemfileVersion ver) dep = dep{dependencyVersion = dependencyVersion dep <|> (Just . CEq) ver}
```

- Remove paren
- Changed to Just (CEq ver) -> (Just . CEq) ver , to keep similar convention to your changes on line 96